src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/RecommendationTabStatus.kt

```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.recommendation
+
+enum class RecommendationTabStatus {
+  CONSIDERING_RECALL, RECOMMENDATION_STARTED, MAKING_DECISION_TO_RECALL, MAKING_DECISION_NOT_TO_RECALL, DECIDED_TO_RECALL, DECIDED_NOT_TO_RECALL, UNKNOWN
```

Comment:
What is 'UNKNOWN' for?

Reply:
Should never happen, but I had to default it to something in the event someone in the future overwrites part of a case and we can't determine the status for whatever reason. Basically we can't test for this so wouldn't worry too much.